.github/workflows/tests.yml

```
@@ -0,0 +1,85 @@
+---
+name: Test collection
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  ansible-lint:
+    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
+  changelog:
+    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+    with:
+      matrix_exclude: >-
```

I just merged https://github.com/ansible-network/github_actions/pull/47 which should let you do this to drop the last stable-2.9 check
```suggestion
      matrix_include: "[]"
      matrix_exclude: >-
```

---

Thank you @Qalthos

---

Oh good, it works!